### ISSUE TITLE
Validation loss doesn't decrease（驗證 loss 不會下降）

### ISSUE THREAD

**Issue by AdrienGuo** [huanglianghua/siamfc-pytorch#58]
This image is the line plot of loss.
Solid line is the loss of train, dashed line is the loss of validation.
The loss of validation doesn't decrease.
<img width="449" alt="Screen Shot 2022-12-27 at 15 08 28" src="https://user-images.githubusercontent.com/50326960/209626446-4654442b-c980-43e8-8dd4-8d09ee5f5ac6.png">
However, the weird thing is, regardless the loss of validation doesn't decrease. The model of 50th epoch is clearly better than the 1st epoch.

Does anyone know this weird phenomenon?

================================================
下圖是我用 GOT-10K 當作資料集的 loss 曲線。
實線是訓練的 loss，虛線是驗證的 loss，驗證 loss 很明顯沒有在下降。
<img width="449" alt="Screen Shot 2022-12-27 at 15 08 28" src="https://user-images.githubusercontent.com/50326960/209626446-4654442b-c980-43e8-8dd4-8d09ee5f5ac6.png">
不過奇怪的是，儘管驗證的 loss 沒有下降，但第50個epoch的model 很明顯比第 1 個還要好很多。

有人可以解釋這種現象嗎？
